Algorithm/SINGON/11047.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+N, K = map(int, input().split())
+cnt = 0
+A = []
+for _ in range(N):
+    X = int(input())
+    A.append(X)
```

Comment:
```suggestion
A = [int(input()) for _ in range(N)]
```
짠! list comprehension 이었습니다.

Reply:
휴 감사합니다. list comprehension이 훨씬 빠르고 좋다는 것은 아는데 습관이 안고쳐지고 까먹게 되네요! 코멘트 감사합니다!

Reply:
오 희락님 드디어 코드리뷰에 눈이 뜨셨군요